lib/src/query_parameters/documents_query.dart

```diff
@@ -6,4 +6,14 @@ class DocumentsQuery extends Queryable {
   final List<String> fields;
 
   DocumentsQuery({this.limit, this.offset, this.fields: const []});
```

Comment:
Why do you need to specify the type for fields?

Reply:
When you define a object-like field, like a list or a real object, the implicit default value is `null`, but since my `fields` is not `List<String>? fields`. I need to specify the value. Or I could make it `required` like this:

```dart
CancelTasksQuery(
      {this.next,
      this.beforeEnqueuedAt,
      this.afterEnqueuedAt,
      this.beforeStartedAt,
      this.afterStartedAt,
      this.beforeFinishedAt,
      this.afterFinishedAt,
      required this.uids, // here
      this.indexUids = const [],
      this.statuses = const [],
      this.types = const []});
```


See more: https://dart.dev/tools/diagnostic-messages?utm_source=dartdev&utm_medium=redir&utm_id=diagcode&utm_content=missing_default_value_for_parameter#missing_default_value_for_parameter